sdk/monitor/arm-monitor/package.json

```diff
@@ -44,7 +44,7 @@
     "chai": "^4.2.0",
     "cross-env": "^7.0.2",
     "@types/node": "^18.0.0",
-    "@azure/arm-eventhub": "5.2.0-beta.2",
+    "@azure/arm-eventhub": "5.2.0-beta.1",
```

Comment:
should this be the workspace version?

Reply:
beta.2 is an unreleased version in sdk repo updated by bot, so I downgrade to the released version beta.1